news/6795.bugfix

```diff
@@ -0,0 +1 @@
+Fix bugs in handling of credentials in URLs.
```

Comment:
I would be more specific by mentioning the case of “a username in the URL with no password (e.g. a token)” since that is the case that everyone has been reporting and would want to know is fixed when reading the release notes. Also, if it’s too vague people might think it might also include the quoting issue, which was also reported but we’re not fixing.

Reply:
Agreed.